plugin/pkg/scheduler/factory/factory.go

```diff
@@ -136,17 +137,58 @@ func NewConfigFactory(
 	// We construct this here instead of in CreateFromKeys because
 	// ScheduledPodLister is something we provide to plug in functions that
 	// they may need to call.
-	var scheduledPodIndexer cache.Indexer
-	scheduledPodIndexer, c.scheduledPodPopulator = cache.NewIndexerInformer(
-		c.createAssignedNonTerminatedPodLW(),
-		&v1.Pod{},
-		0,
-		cache.ResourceEventHandlerFuncs{
-			AddFunc:    c.addPodToCache,
-			UpdateFunc: c.updatePodInCache,
-			DeleteFunc: c.deletePodFromCache,
+	scheduledPodIndexer := podInformer.Informer().GetIndexer()
+	scheduledPodIndexer.AddIndexers(cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
+	c.scheduledPodPopulator = podInformer.Informer().GetController()
```

Comment:
this is suspect.  Some controller calls don't work right on informers anymore.  I remember wiring them to dummy.